src/main/java/graphql/analysis/QueryTraverser.java

```diff
@@ -70,7 +72,7 @@ private QueryTraverser(GraphQLSchema schema,
         this.fragmentsByName = getOperationResult.fragmentsByName;
         this.roots = singletonList(getOperationResult.operationDefinition);
         this.rootParentType = getRootTypeFromOperation(getOperationResult.operationDefinition);
-        this.coercedVariables = ValuesResolver.coerceVariableValues(schema, variableDefinitions, rawVariables);
+        this.coercedVariables = ValuesResolver.coerceVariableValues(schema, variableDefinitions, rawVariables, GraphQLContext.getDefault(), Locale.getDefault());
```

Comment:
This entry point come not from a graphql request and hence uses a default graphql context and default locale.  Perhaps in some future change these are passed in.